drivers/ws281x/Makefile.dep

```diff
@@ -28,3 +31,10 @@ ifneq (,$(filter ws281x_esp32%,$(USEMODULE)))
     USEMODULE += ws281x_esp32_hw
   endif
 endif
+
+ifneq (,$(filter ws281x_spi,$(USEMODULE)))
+  FEATURES_REQUIRED += periph_spi
+  FEATURES_REQUIRED += periph_dma
```

Comment:
```suggestion
  FEATURES_OPTIONAL += periph_dma
```

would also suffice - I'm also not sure about unconditionally enabling DMA as we have no way of selectively enabling DMA - it's always enabled for everything.